src/core/Akka.Remote/Transport/Helios/HeliosTcpTransport.cs

```diff
@@ -106,8 +106,19 @@ public override void ChannelRead(IChannelHandlerContext context, object message)
         public override void ExceptionCaught(IChannelHandlerContext context, Exception exception)
         {
             var se = exception as SocketException;
+
             if (se?.SocketErrorCode == SocketError.OperationAborted)
             {
+                Log.Info("Socket read operation aborted. Connection is about to be closed. Channel [{0}->{1}](Id={2})",
+                    context.Channel.LocalAddress, context.Channel.RemoteAddress, context.Channel.Id);
+
+                NotifyListener(new Disassociated(DisassociateInfo.Shutdown));
+            }
+            else if (se?.SocketErrorCode == SocketError.ConnectionReset)
```

Comment:
New case is here. When it happens, we log the warning and sending DisassociateInfo.Shutdown.